client/post-editor/edit-post-status/index.jsx

```diff
@@ -52,48 +57,50 @@ export default React.createClass( {
 			stickyEventLabel = 'Off';
 		}
 
-		stats.recordStat( stickyStat );
-		stats.recordEvent( 'Changed Sticky Setting', stickyEventLabel );
+		recordStat( stickyStat );
+		recordEvent( 'Changed Sticky Setting', stickyEventLabel );
 
-		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
-		actions.edit( { sticky: ! this.props.post.sticky } );
-	},
+		this.props.editPost( this.props.siteId, this.props.postId,
```

Comment:
I suppose it could be personal preference, but stylistically this sticks out, partly because of the inconsistency between the first two arguments being on the same line, then the third on its own own. Personally I'd have written as:

```js
this.props.editPost( this.props.siteId, this.props.postId, {
	sticky: ! this.props.post.sticky
} );
```